src/test/architecture/test_architecture_loader.py

```diff
@@ -418,7 +418,7 @@ def assigned(node_paths, message_contexts):
         assert node.timers == [timer]
         assert node.subscriptions == [subscription]
         assert node.services == [service]
-        assert node.callbacks == [callback]
+        assert node.callbacks == [service.callback, subscription.callback, timer.callback]
```

Comment:
`service.callback` is Mock.
It must be configured to return service_callback in advance as follows.
```
service_callback = mocker.Mock(spec=ServiceCallbackStruct)
subscription_callback = mocker.Mock(spec=SubscriptionCallbackStruct)
timer_callback = mocker.Mock(spec=TimerCallbackStruct)
mocker.patch.object(subscription, 'callback', service_callback)
mocker.patch.object(service, 'callback', subscription_callback)
mocker.patch.object(timer, 'callback', timer_callback)
```

Reply:
Fixed in https://github.com/tier4/caret_analyze/pull/506/commits/5a9871e13ca3a5ee646d814c9abe37187cad6526.